config/Database.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+class Database
+{
+    private $host = 'localhost';
+    private $user = 'root';
+    private $password = '';
+    private $dbname = 'cms';
+    private $port = 4000;
```

Comment:
This port you added here made this application not work on my local machine. By default mysql uses 3306. Why is yours using 4000? Cause here you are assuming that everyone using your application will work with port 4000, hence it will throw an error on those using a port other than 4000.

So, you need to look for a fix for this.